tests/queries/0_stateless/03100_lwu_40_cleanup_race.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
```

Comment:
This new test seems flaky, see https://play.clickhouse.com/play?user=play#c2VsZWN0IAp0ZXN0X25hbWUsIGNoZWNrX3N0YXJ0X3RpbWUsIHB1bGxfcmVxdWVzdF9udW1iZXIsIHJlcG9ydF91cmwKZnJvbSBjaGVja3Mgd2hlcmUgJzIwMjUtMDgtMDEnIDw9IGNoZWNrX3N0YXJ0X3RpbWUgYW5kIHRlc3RfbmFtZSBsaWtlICclMDMxMDBfbHd1XzQwX2NsZWFudXBfcmFjZSUnIGFuZCB0ZXN0X3N0YXR1cyBpbiAoJ0ZBSUwnLCAnRkxBS1knKSBvcmRlciBieSBjaGVja19zdGFydF90aW1l